packages/jaeger-ui-components/src/scroll-page.test.ts

```diff
@@ -115,16 +116,19 @@ describe('scroll-by', () => {
       expect(tweenInstances.length).toBe(1);
       const tw = tweenInstances[0];
       cancel();
-      expect(tw.cancel.mock.calls).toEqual([[]]);
-      tw.cancel.mockReset();
+      expect(jest.mocked(tw.cancel).mock.calls).toEqual([[]]);
+      jest.mocked(tw.cancel).mockReset();
       // now, we check to see if `cancel()` has an effect on the last created tween
       cancel();
-      expect(tw.cancel.mock.calls.length).toBe(0);
+      expect(jest.mocked(tw.cancel).mock.calls.length).toBe(0);
     });
   });
 
   describe('_onTweenUpdate', () => {
-    let oldScrollTo;
+    let oldScrollTo: { (options?: ScrollToOptions | undefined): void; (x: number, y: number): void } & {
+      (options?: ScrollToOptions | undefined): void;
+      (x: number, y: number): void;
+    };
```

Comment:
```suggestion
    let oldScrollTo: { (options?: ScrollToOptions | undefined): void; (x: number, y: number): void; };
```

Reply:
Done in commit https://github.com/grafana/grafana/pull/59617/commits/5630dd2f99edc0add1a20e31e871608f88803023.